packages/client/src/library/Svg/AccordionClosed.js

```diff
@@ -0,0 +1,19 @@
+const AccordionClosed = ({ width = "16", height = "17", className = "" }) => (
```

Comment:
Suggest renaming to CaretRight for readability and icon naming consistency

Reply:
Renamed